node/Cargo.toml

```diff
@@ -81,6 +81,11 @@ cumulus-relay-chain-inprocess-interface = { git = 'https://github.com/paritytech
 cumulus-relay-chain-interface = { git = 'https://github.com/paritytech/cumulus.git', branch = "polkadot-v0.9.26" }
 cumulus-relay-chain-rpc-interface = { git = 'https://github.com/paritytech/cumulus.git', branch = "polkadot-v0.9.26" }
 
+# Nimbus dependencies
+nimbus-consensus = { git = "https://github.com/manta-network/nimbus.git", branch = "garandor/polkadot-v0.9.26", default-features = false }
+nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", branch = "garandor/polkadot-v0.9.26", default-features = false }
+pallet-author-inherent = { git = "https://github.com/manta-network/nimbus.git", branch = "garandor/polkadot-v0.9.26", default-features = false }
```

Comment:
Don't think you need the `default-features = false` here.

Reply:
Thanks, this was bad copypasta